src/widgets/conditions/conditions-detailed-summary.css

```diff
@@ -0,0 +1,108 @@
+:global(.omrs-breakpoint-gt-phone) .conditionSummary {
+  display: flex;
+  justify-content: center;
+}
+
+:global(.omrs-breakpoint-lt-tablet) .conditionSummary {
+  padding: 0.25rem 0.5rem 0 0.5rem;
+}
+
+.conditionTable {
+  width: 100%;
+}
+
+.conditionTable thead {
+  color: var(--omrs-color-ink-medium-contrast);
+  border-bottom: 1px solid var(--omrs-color-bg-low-contrast);
+}
+
+.conditionTable thead td {
+  padding: 1rem 0.25rem 1rem 1rem;
+}
+
+:global(.omrs-breakpoint-lt-tablet) .conditionTable thead td {
+  padding: 0.25rem 0.125rem 0.5rem 0.5rem;
+}
+
+.conditionTable thead td:nth-child(2) {
+  text-align: right;
+  padding-right: 0rem;
+  background: "lightgray";
```

Comment:
```suggestion
  background: var(--omrs-color-ink-low-contrast);
```

Reply:
That background property shouldn't be there. I was using it for testing. I've removed it.